samples/bluetooth/peripheral_status/README.rst

```diff
@@ -0,0 +1,129 @@
+.. _peripheral_status:
+
+Bluetooth: Peripheral Status
+############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+
+The peripheral Status sample demonstrates how to use the :ref:`nsms_readme`.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+.. include:: /includes/tfm.txt
+
+The sample also requires a smartphone or tablet running a compatible application.
+The `Testing`_ instructions refer to `nRF Connect for Mobile`_, but you can also use other similar applications (for example, `nRF Toolbox`_).
+
+.. note::
+   |thingy53_sample_note|
+
+Overview
+********
+
+Sample implements two NSMS instances that results in two NSMS characteristics that describes status of two buttons (button1 and button2).
+Each characteristic have name set in Characteristic User Description (CUD) that coresponds to the button whose status is shown.
+
+Moreover status related to button 2 have configurable security settings.
+When :kconfig:option:`CONFIG_BT_STATUS_SECURITY_ENABLED` is configured status related to button 2 have security enabled thous it can be accessed only when bonded.
```

Comment:
```suggestion
Moreover, status related to **Button 2** has configurable security settings.
When :kconfig:option:`CONFIG_BT_STATUS_SECURITY_ENABLED` is configured, the status related to **Button 2** has security enabled, thus it can be accessed only when bonded.
```